ironic-config/ironic.conf.j2

```diff
@@ -191,11 +186,29 @@ insecure = {{ env.IRONIC_INSECURE }}
 [nova]
 send_power_notifications = false
 
+# Sections (oslo_messaging_notifications, sensor_data, metrics) required for sensor data collection using ironic-prometheus-exporter (IPE):
+{% if env.SEND_SENSOR_DATA | lower == "true" %}
 [oslo_messaging_notifications]
 driver = prometheus_exporter
 location = /shared/ironic_prometheus_exporter
 transport_url = fake://
 
+[sensor_data]
+send_sensor_data = {{ env.SEND_SENSOR_DATA }}
+# NOTE(TheJulia): Do not lower this value below 120 seconds.
+# Power state is checked every 60 seconds and BMC activity should
+# be avoided more often than once every sixty seconds.
+# NOTE: When moving this change to downstream (openshift), this must be set to 60 or less.
+interval = {{ env.SENSOR_DATA_INTERVAL | default('160') }}
+# Additional sensor_data options can be configured via OS_ environment variables:
+# OS_SENSOR_DATA__WORKERS, OS_SENSOR_DATA__WAIT_TIMEOUT, OS_SENSOR_DATA__DATA_TYPES,
+# OS_SENSOR_DATA__ENABLE_FOR_UNDEPLOYED_NODES, OS_SENSOR_DATA__ENABLE_FOR_CONDUCTOR,
+# OS_SENSOR_DATA__ENABLE_FOR_NODES
```

Comment:
better link to https://docs.openstack.org/ironic/latest/configuration/config.html#sensor-data